### PR TITLE
[FEMDEM] Solving memory error and update tests to 1 thread

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
That's it

### CODE REVIEW

**Review by joaquinirazabal — APPROVED** [review]
Great, @AlejandroCornejo, it is working fine for me :)

File: applications/FemToDemApplication/tests/main_coupling_for_testing_tables.py [hunk 1]
```diff
@@ -13,6 +13,11 @@ def KratosPrintInfo(message):
     KratosMultiphysics.Logger.Print(message, label="")
     KratosMultiphysics.Logger.Flush()
 
+def IsOk(value, ref, tol):
+    if abs(value - ref) / ref > tol:
+        return False
+    return True
```
Comment (philbucher) [inline applications/FemToDemApplication/tests/main_coupling_for_testing_tables.py:19]: I recommend using the functionalities procided by pythons unittest module

File: applications/FemToDemApplication/tests/main_coupling_free_fall_testing.py [hunk 2]
```diff
@@ -13,45 +13,56 @@ def KratosPrintInfo(message):
     KratosMultiphysics.Logger.Print(message, label="")
     KratosMultiphysics.Logger.Flush()
 
+def IsOk(value, ref, tol):
+    if abs(value - ref) / ref > tol:
+        return False
+    return True
+
```
Comment (philbucher) [inline applications/FemToDemApplication/tests/main_coupling_free_fall_testing.py:20]: same

File: applications/FemToDemApplication/tests/main_coupling_total_lagrangian_for_testing.py [hunk 3]
```diff
@@ -15,43 +15,52 @@ def KratosPrintInfo(message):
     KratosMultiphysics.Logger.Print(message, label="")
     KratosMultiphysics.Logger.Flush()
 
+def IsOk(value, ref, tol):
+    if abs(value - ref) / ref > tol:
+        return False
+    return True
```
Comment (philbucher) [inline applications/FemToDemApplication/tests/main_coupling_total_lagrangian_for_testing.py:21]: same

**Review by philbucher — COMMENTED** [review]
minor

**Pull request merged**